examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in

```diff
@@ -0,0 +1,17 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
```

Comment:
could you also pls remove "converter_" from the filename of all of those converter examples. The folder has that information in them

Reply:
Yes, I'll do this.